.github/workflows/update-web-mod-definitions.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     name: Update osu-web mod definitions
     runs-on: ubuntu-latest
     steps:
-    - name: Install .NET 6.0.x
+    - name: Install .NET 8.0.x
```

Comment:
This means that `osu-tools` needs an update to .NET 8 too, or this job is going to start failing due to a .NET 6 project (`osu-tools`) attempting to locally reference a .NET 8 project (the game).

Reply:
Done in https://github.com/ppy/osu-tools/pull/196